libtextsecure/message_receiver.js

```diff
@@ -1261,7 +1235,7 @@ MessageReceiver.prototype.extend({
       );
     }
 
-    return libloki.savePreKeyBundleForNumber({
+    await libloki.savePreKeyBundleForNumber({
```

Comment:
Technically returning an async function call implicitly awaits it: `return await foo()` is the same as `return foo()`.
So here you just skipped returning a value, not sure if necessary?

Reply:
I think I like the await to make it more explicit that the async function is just resolving rather than returning a value that we want to use?

Reply:
As discussed offline, I agree it's good to `return` only when there is something to return :)